queue/queue.py

```diff
@@ -3,13 +3,29 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = [] 
 
   def enqueue(self, item):
-    pass
+    self.storage.insert(self.size, item)
+    self.size += 1
   
   def dequeue(self):
-    pass
+        if self.size > 0:
+
+            self.size -= 1
+            return self.storage.pop()
+        return None
 
   def len(self):
-    pass
+      return self.size
+
```

Comment:
Overall, excellent job! Remember, your `len(self)` is going to be a constant time operation, as you are just returning the size counter!